compiler/noirc_evaluator/src/ssa/opt/inline_simple_functions.rs

```diff
@@ -34,8 +39,8 @@ impl Ssa {
                 return false;
             }
 
-            // Only inline functions with 0 or 1 instructions
-            if instructions.len() > 1 {
+            // Only inline functions with 10 or less instructions
+            if instructions.len() > 10 {
```

Comment:
Should we make this configurable in the `SsaEvaluatorOptions`, seeing that the inliner aggressiveness is configurable as well? It would avoid a magic number in the code.
Also leaving some hints about how this sweet spot has been determined (even if it's just an expert guess) would help auditors and future devs.

Reply:
I will make some more comments about this number and move it into a constant global. We could make this configurable, although I'm a little hesitant as this pass is a bit of a workaround of our inliner cost analysis and aggressiveness setting not already marking these functions to be inlined. Improving the inliner cost analysis is a larger task in its own so I think have this kind of inlining override is ok for now though.

Reply:
done